src/typography/index.md

```diff
@@ -25,15 +25,17 @@ The Government Digital Service adapted it in 2012 for digital use, bringing the
 
 Using GDS Transport makes GOV.UK easier to recognise and information easier to read on any device.
 
-![TODO](./transport-font.svg)
+<div class="scale width-80 right edge">
+
+![Sample of GDS Transport font in bold and regular weight](./transport-font.svg)
 
 {% endsectionHighlight %}
 
 ### Weights
 
-GDS transport consists of two weights; **Light** and **Bold**.
+GDS transport consists of two weights: light and bold.
```

Comment:
Also just noticed the lowercase "t" on "transport" which should be uppercase.

Reply:
Fixed in recent commit.